spec/features/edit_organisation_details_spec.rb

```diff
@@ -32,4 +32,17 @@
       }.to raise_error(ActionController::RoutingError)
     end
   end
+
+  context 'when inputting invalid details' do
+    before do
+      sign_in_user user
+      visit edit_organisation_path(organisation)
+      fill_in 'Service email', with: ''
+      click_on 'Save'
+    end
+
+    it 'will cause routing error' do
```

Comment:
Definitely being pedantic here but `"it will cause routing error"`, isn't whats being asserted here. The assertion is that the page shows the error message, so a better spec description would be `it "displays an error message to the user"`

Reply:
Yeah makes sense, updated